src/script_chainer/config/script_config.py

```
@@ -36,32 +37,46 @@ class GameProcessName(Enum):
+    kill_script_after_done: bool = True
+    kill_game_after_done: bool = True
+    script_arguments: str = ''
+    notify_start: bool = True
+    notify_done: bool = True
+    enabled: bool = True
+
+    # 不参与序列化的元数据
+    idx: int = field(default=0, repr=False, compare=False)
+
+    def to_dict(self) -> dict:
+        """序列化为字典（排除 idx）。"""
```

Ruff (0.15.9)

[warning] Docstring contains ambiguous `（` (FULLWIDTH LEFT PARENTHESIS). Did you mean `(` (LEFT PARENTHESIS)?

(RUF002)